src/commons/error.rs

```diff
@@ -273,6 +276,13 @@ pub enum Error {
     AspaProvidersUpdateEmpty(CaHandle, AspaCustomer),
     AspaProvidersUpdateConflict(CaHandle, AspaProvidersUpdateConflict),
 
+    //-----------------------------------------------------------------
+    // BGP Sec
+    //-----------------------------------------------------------------
+    BgpSecDefinitionUnknown(CaHandle, BgpSecAsnKey),
+    BgpSecDefinitionInvalidlySigned(CaHandle, BgpSecDefinition),
```

Comment:
It was "invalidly signed". The signature (which in itself may be a fine specimen) does not match the content. It could be called "InvalidSignature" but it's not shorter, nor better imho. So, I am leaving this as is.

Perhaps we will add more detail when the better error reporting is merged into rpki-rs.